Exploration Queries/IoT/Process_byIoTDevice.yaml

```diff
@@ -0,0 +1,24 @@
+﻿Id: 1f3ecde7-5c69-4d44-ac93-5feac6d1cd2f
+DisplayName: Most frequent command executions on the device
+Description: Most frequent command executions on the device
+InputEntityType: IoTDevice
+InputFields:
+  - DeviceId
+  - IoTHub
+OutputEntityTypes:
+  - Process
+QueryPeriodBefore: 30m
+QueryPeriodAfter: 30m
+DataSources: SecurityIoTRawEvent
+query: |
+
+  let Process_byIoTDevice = (v_IotDevice_DeviceId:string, v_IoTDevice_IoTHub:string){
+  SecurityIoTRawEvent 
+  | where RawEventName == 'ProcessCreate'
+  | where AssociatedResourceId == parse_json(v_IoTDevice_IoTHub)['ResourceId'] and DeviceId == v_IotDevice_DeviceId
+  | extend Process_CommandLine = tostring(parse_json(EventDetails)['CommandLine'])
+  | summarize procCount = count() by Process_CommandLine
+  | top 10 by procCount 
+  | project-away procCount
```

Comment:
I don't think we should be projecting away the process count, this is useful information for sorting in the investigation graph.

Reply:
Hi Shain, how should I project this field so that Sentinel can make use of it?

Reply:
My fault, had regular query review on my mind, no need for this in the exploration queries.